src/main/java/com/github/lunatrius/schematica/client/gui/control/GuiSchematicMaterialsSlot.java

```diff
@@ -44,6 +49,22 @@ protected void drawBackground() {
     protected void drawContainerBackground(Tessellator tessellator) {
     }
 
+    /**
+     * Scan your inventory item counts
+     */
+    protected void updateCount() {
```

Comment:
`updateInventoryAmount` is less ambiguous. I'll have to check the 1.7.10 version, but there should already be a method in LunatriusCore that counts inventory items.